src/webapp/components/survey-list/SurveyList.tsx

```diff
@@ -1,167 +1,108 @@
 import i18n from "@eyeseetea/feedback-component/locales";
-import {
-    Button,
-    Paper,
-    TableCell,
-    TableContainer,
-    TableHead,
-    Typography,
-    Table,
-    TableBody,
-    TableRow,
-} from "@material-ui/core";
-import { NavLink, useHistory } from "react-router-dom";
+import { Button, Typography } from "@material-ui/core";
+import { NavLink } from "react-router-dom";
 import styled from "styled-components";
 import { Id } from "../../../domain/entities/Ref";
 import { useSurveys } from "../../hooks/useSurveys";
 import { palette } from "../../pages/app/themes/dhis2.theme";
-import { ActionMenuButton } from "../action-menu-button/ActionMenuButton";
-import { SURVEY_FORM_TYPES } from "../../../domain/entities/Survey";
+import { SurveyBase, SURVEY_FORM_TYPES } from "../../../domain/entities/Survey";
 import { CustomCard } from "../custom-card/CustomCard";
-import { useEffect, useState } from "react";
 import { useCurrentSurveys } from "../../contexts/current-surveys-context";
 import { ContentLoader } from "../content-loader/ContentLoader";
+import { getSurveyDisplayName, hideCreateNewButton } from "../../../domain/utils/PPSProgramsHelper";
+import { getUserAccess } from "../../../domain/utils/menuHelper";
+import { useAppContext } from "../../contexts/app-context";
+import { useCurrentModule } from "../../contexts/current-module-context";
+import { SurveyListTable } from "./SurveyListTable";
+import { SurveyListFilters } from "./SurveyListFilters";
+import _ from "../../../domain/entities/generic/Collection";
+import { useSurveyList } from "./hook/useSurveyList";
 
 interface SurveyListProps {
-    surveyType: SURVEY_FORM_TYPES;
+    surveyFormType: SURVEY_FORM_TYPES;
 }
-export const SurveyList: React.FC<SurveyListProps> = ({ surveyType }) => {
-    const { currentPPSSurveyForm, changeCurrentPPSSurveyForm } = useCurrentSurveys();
-    const { surveys, loading, error } = useSurveys(surveyType, currentPPSSurveyForm);
-    const [options, setOptions] = useState<string[]>([]);
-    const history = useHistory();
+export const SurveyList: React.FC<SurveyListProps> = ({ surveyFormType }) => {
+    const {
+        currentPPSSurveyForm,
+        changeCurrentPPSSurveyForm,
+        changeCurrentCountryQuestionnaire,
+        changeCurrentHospitalForm,
+        changeCurrentWardRegister,
+    } = useCurrentSurveys();
+    const { currentUser } = useAppContext();
+    const { currentModule } = useCurrentModule();
 
-    useEffect(() => {
-        if (currentPPSSurveyForm) {
-            setOptions(["Edit"]);
-        } else {
-            setOptions(["Edit", "Assign Country", "List Countries"]);
-        }
-    }, [setOptions, currentPPSSurveyForm]);
+    let isAdmin = false;
```

Comment:
`const isAdmin = currentModule ? getUserAccess(currentModule, currentUser.userGroups).hasAdminAccess : false;`

Reply:
Done in https://github.com/EyeSeeTea/amr-surveys/pull/11